doc/getting_started.rst

```diff
@@ -419,4 +421,4 @@ The autoscale group will start destroying all your servers. Now you can fire a D
   curl --include \
      --header "X-Auth-token: {auth-token}" \
      --request DELETE \
-     "https://ord.autoscale.api.rackspacecloud.com/v1.0/{tenantId}/groups/{groupId}"
\ No newline at end of file
+     "https://ord.autoscale.api.rackspacecloud.com/v1.0/{tenantId}/groups/{groupId}"
```

Comment:
GitHub usually shows it, but it's because there was no newline at the end of the file, which my editor automatically adds. I verified this is the actual change using git on the command line to look at the diff.